app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="post-form">
+  <div class="card new-post">
+    <div class="card-body">
+      <h2>Forgot your password?</h2>
+
+      <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+        <%= render "devise/shared/error_messages", resource: resource %>
+
+        <div class="field mb-3">
+          <%= f.label :email %><br />
+          <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control" %>
+        </div>
+
+        <div class="actions mb-3">
+          <%= f.submit "Send me reset password instructions" %>
+        </div>
+      <% end %>
+
+      <%= render "devise/shared/links" %>
+    </div>
+  </div>
+</div>
```

Comment:
- [ ] User should be able to reset a password in case he forgot but am getting a server error `500` please work on that